cmd/av/stack_sync.go

```diff
@@ -327,24 +320,55 @@ base branch.
 			})
 
 			parentMeta, ok := branches[parentState.Name]
-			if ok && parentMeta.PullRequest != nil && parentMeta.PullRequest.State == githubv4.PullRequestStateMerged {
+			if ok && parentMeta.TrunkCommit != nil {
 				defaultBranch, err := repo.DefaultBranch()
```

Comment:
We should actually figure out the name of the trunk branch when we start the sync and not assume it's the default branch name here. But probably outside the scope of this PR.